modules/end-user-guide/partials/proc_mounting-a-secret-as-an-environment-variable-into-a-workspace-container.adoc

```diff
@@ -5,19 +5,81 @@
 [id="mounting-a-secret-as-an-environment-variable-into-a-workspace-container_{context}"]
 = Mounting a secret as an environment variable into a workspace container
 
-The following section describes how to mount a {platforms-name} secret from the user’s {orch-namespace} as an environment variable, or variables, into single-workspace or multiple-workspace containers of {prod-short}.
+A secret can be mounted from the user’s {orch-namespace} as an environment variable (or variables) in single-workspace or multiple-workspace containers of {prod-short}.
```

Comment:
Extend the suggestions from the previous topic to this one.

Reply:
@rolfedh Thank you, Rolfe!